src/main/java/yevhenii/lostfilmdemo/rss/DocumentParser.java

```diff
@@ -0,0 +1,57 @@
+package yevhenii.lostfilmdemo.rss;
+
+import org.springframework.stereotype.Component;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+import yevhenii.lostfilmdemo.services.FeedService;
+
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class DocumentParser implements FeedService {
+    private final List<FeedMessage> messages = new ArrayList<>();
+
+    Document readDocument(String url) throws ParserConfigurationException, IOException, SAXException {
```

Comment:
add `private` access modification, it's not mandatory but with that code looks nicer